test/api/v3/unit/libs/payments.test.js

```diff
@@ -3,15 +3,27 @@ import * as api from '../../../../../website/server/libs/payments';
 import analytics from '../../../../../website/server/libs/analyticsService';
 import notifications from '../../../../../website/server/libs/pushNotifications';
 import { model as User } from '../../../../../website/server/models/user';
+import { model as Group } from '../../../../../website/server/models/group';
 import moment from 'moment';
+import {
+  generateGroup,
+} from '../../../../helpers/api-unit.helper.js';
 
 describe('payments/index', () => {
-  let user, data, plan;
+  let user, group, data, plan;
 
-  beforeEach(() => {
+  beforeEach(async () => {
     user = new User();
     user.profile.name = 'sender';
 
+    group = generateGroup({
```

Comment:
you're right, I got confused with the helper named the same way but for integration tests